src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -26,12 +28,25 @@ public static Elevator getInstance() {
 
     private Elevator() { 
         super(Constants.Elevator.RIGHT_MOTOR, Constants.Elevator.LEFT_MOTOR, 
-        Constants.Elevator.FF_CONFIG, false, true, "elevator");   
+        Constants.Elevator.FF_CONFIG, Constants.Elevator.TUNNING_MODE, true, "elevator");   
         
+        scoringPos = new TunableNumber("Elevator Scoring Position", 0, true);
         limitSwitch = new DigitalInput(0);
         resetSequence = false; 
     }
 
+    public Command scoreGamepiece() {
+        if (scoringPos.get() < 1 || scoringPos.get() > 3) {
+            System.out.println("please give an actual scoring location");
+            return runOnce(() -> {});   // hope that works
+        } else {
+            Manipulator manipulator = Manipulator.getInstance();
```

Comment:
subsytems shoudn't house other subsystems if needed create a command file,  for now tho just move this functionality to robot container